.github/workflows/test.yaml

```diff
@@ -65,7 +69,81 @@ jobs:
             ~/go/pkg/mod
             ~/.cache/go-build
       - uses: crazy-max/ghaction-github-runtime@v1
-      - name: Build main
+      - name: Build controller
         run: make docker-build CACHE_FROM=type=gha,scope=eraser-manager CACHE_TO=type=gha,scope=eraser-manager,mode=max
+
       - name: Build eraser
         run: make docker-build-eraser CACHE_FROM=type=gha,scope=eraser-node CACHE_TO=type=gha,scope=eraser-node,mode=max
+
+      - name: Pull test image
+        run: docker pull hello-world
+
+      - name: Create k8s Kind Cluster
+        uses: helm/kind-action@v1.2.0
+        with:
+          cluster_name: ${{ env.CLUSTER_NAME }}
+          config: .github/workflows/kind-config.yaml
+      - run: kubectl get pod -A
+      - run: kubectl get nodes -A
+
+      - name: Load controller and eraser image on the nodes of the cluster
+        run: |
+          kind load docker-image aldaircoronel/eraser-manager:test --name ${{ env.CLUSTER_NAME }}
+          kind load docker-image aldaircoronel/eraser:test --name ${{ env.CLUSTER_NAME }}
+          kind load docker-image hello-world:latest --name ${{ env.CLUSTER_NAME }}
```

Comment:
these can be combined into a single line `kind load docker-image hello-world eraser:test eraser-manager:test ...`